stories/types.stories.tsx

```diff
@@ -24,6 +25,34 @@ const props = {
   };
 
 storiesOf('Types', module)
+  .add('Form Stress Test (all)', () => (
+    <FormCard
+      {...formCardPropsFactory.build()}
+      {...omit(props, 'fieldSets')}
+      fieldSets={(
+        // tslint:disable-next-line no-magic-numbers
+        Array(50)
+          .fill(props.fieldSets[0])
+          .map((fieldSet, idx) => (
+            fieldSet.map((fieldConfig: IFieldConfigPartial) => ({
+              ...fieldConfig,
+              field: fieldConfig.field + idx,
+            }))
+          ))
+      )}
+    />
+  ))
+  .add('Form Stress Test (text)', () => (
+    <FormCard
+      {...formCardPropsFactory.build()}
+      {...omit(props, 'fieldSets')}
+      fieldSets={[
+        // tslint:disable-next-line no-magic-numbers
+        Array(159)
```

Comment:
I realize where this is from, but we should probably have it be a more round number for it to be less magical

Reply:
Hmmm... I could raise it to 160 but I think 200 would be too high and would make it no longer representative. Since this is our largest known use-case It's not that magical? I could lower it to 150 but then it kinda loses it's usefulness.

Reply:
I'd say make it 175, especially because our use-case is dynamic